+nosnoc/+solver/MpccSolver.m

```diff
@@ -359,11 +692,10 @@
                     opts_casadi_nlp.ipopt.dual_inf_tol = default_tol;
                     opts_casadi_nlp.ipopt.compl_inf_tol = default_tol;
                     opts_casadi_nlp.ipopt.resto_failure_feasibility_threshold = 0;
-                    opts_casadi_nlp.ipopt.print_level = obj.opts.opts_casadi_nlp.ipopt.print_level;
+                    opts_casadi_nlp.ipopt.print_level = 5;%obj.opts.opts_casadi_nlp.ipopt.print_level;
                     opts_casadi_nlp.print_time = obj.opts.opts_casadi_nlp.print_time;
                     opts_casadi_nlp.ipopt.sb = 'yes';
-                    
-                    
+                   
                     tnlp_solver = nlpsol('tnlp', 'ipopt', casadi_nlp, opts_casadi_nlp);
```

Comment:
so independent which plugin is choose, tnlp is always via ipopt (fine for me, would be pain in the ass to do others, with no value added to it)

Reply:
Yea theoretically we can make it the same as the main loop, but I have had better performance with ipopt than others when I was testing this for my thesis.